lib/src/generator.dart

```diff
@@ -36,9 +38,26 @@ class InvalidGenerationSourceError extends Error {
   /// May be an empty string if unknown.
   final String todo;
 
-  InvalidGenerationSourceError(this.message, {String todo})
+  /// The code element associated with this error.
+  ///
+  /// May be `null` if the error had no associated element.
+  final Element associatedElement;
```

Comment:
I'm not sure if the word "associated" is making this more clear than it would be with only "element"